CMake/hoomd/HOOMDMPISetup.cmake

```diff
@@ -9,8 +9,13 @@ if (ENABLE_MPI)
     if (cereal_FOUND)
         find_package_message(cereal "Found cereal: ${cereal_DIR}" "[${cereal_DIR}]")
     else()
+        # work around missing ceralConfig.cmake
+
+        find_path(cereal_INCLUDE_DIR NAMES cereal/cereal.hpp
+            PATHS ${CMAKE_INSTALL_PREFIX}/include)
         add_library(cereal INTERFACE IMPORTED)
-        find_package_message(cereal "Could not find cereal, assuming it is on a default path" "[${cereal_DIR}]")
+        target_include_directories(cereal INTERFACE ${cereal_INCLUDE_DIR})
```

Comment:
```suggestion
        set_target_properties(cereal PROPERTIES
            INTERFACE_INCLUDE_DIRECTORIES "${cereal_INCLUDE_DIR}")
```

Reply:
> How did you install cereal in such a way that `ceralConfig.cmake` is missing? That workaround was intended for platforms like Ubuntu that fails to package `cerealConfig.cmake` for unknown reasons.

```conda install -c conda-forge cereal```

Reply:
Thanks, that conda package should be improved to install the cmake files: conda-forge/cereal-feedstock#1